iset.mm

```diff
@@ -135016,6 +135038,642 @@ the base set to the (extended) reals and which is nonnegative, symmetric
+
+  ${
+    msf.x $e |- X = ( Base ` M ) $.
+    msf.d $e |- D = ( ( dist ` M ) |` ( X X. X ) ) $.
+    $( The distance function, suitably truncated, is a metric on ` X ` .
+       (Contributed by Mario Carneiro, 2-Sep-2015.) $)
+    xmsxmet $p |- ( M e. *MetSp -> D e. ( *Met ` X ) ) $=
+      ( cxms wcel cxmet cfv ctopn cmopn wceq eqid isxms2 simplbi ) BFGACHIGBJIZ
+      AKILAPBCPMDENO $.
+
+    $( The distance function, suitably truncated, is a metric on ` X ` .
+       (Contributed by Mario Carneiro, 12-Nov-2013.) $)
+    msmet $p |- ( M e. MetSp -> D e. ( Met ` X ) ) $=
+      ( cms wcel cmet cfv ctopn cmopn wceq eqid isms2 simplbi ) BFGACHIGBJIZAKI
+      LAPBCPMDENO $.
+
+    $( Mapping of the distance function of a metric space.  (Contributed by NM,
+       30-Aug-2006.)  (Revised by Mario Carneiro, 12-Nov-2013.) $)
+    msf $p |- ( M e. MetSp -> D : ( X X. X ) --> RR ) $=
+      ( cms wcel cmet cfv cxp cr wf msmet metf syl ) BFGACHIGCCJKALABCDEMACNO
+      $.
+  $}
+
+  ${
+    mscl.x $e |- X = ( Base ` M ) $.
+    mscl.d $e |- D = ( dist ` M ) $.
+    $( The distance function, suitably truncated, is a metric on ` X ` .
+       (Contributed by Mario Carneiro, 2-Oct-2015.) $)
+    xmsxmet2 $p |- ( M e. *MetSp -> ( D |` ( X X. X ) ) e. ( *Met ` X ) ) $=
+      ( cxp cres cds cfv reseq1i xmsxmet ) ACCFZGBCDABHILEJK $.
+
+    $( The distance function, suitably truncated, is a metric on ` X ` .
+       (Contributed by Mario Carneiro, 2-Oct-2015.) $)
+    msmet2 $p |- ( M e. MetSp -> ( D |` ( X X. X ) ) e. ( Met ` X ) ) $=
+      ( cxp cres cds cfv reseq1i msmet ) ACCFZGBCDABHILEJK $.
+
+    $( Closure of the distance function of a metric space.  (Contributed by NM,
+       30-Aug-2006.)  (Revised by Mario Carneiro, 2-Oct-2015.) $)
+    mscl $p |- ( ( M e. MetSp /\ A e. X /\ B e. X ) -> ( A D B ) e. RR ) $=
+      ( cms wcel w3a cxp cres co cr wceq ovres 3adant1 cmet cfv msmet2 eqeltrrd
+      metcl syl3an1 ) DHIZAEIZBEIZJABCEEKLZMZABCMZNUEUFUHUIOUDABEECPQUDUGERSIUE
+      UFUHNICDEFGTABUGEUBUCUA $.
+
+    $( Closure of the distance function of an extended metric space.
+       (Contributed by Mario Carneiro, 2-Oct-2015.) $)
+    xmscl $p |- ( ( M e. *MetSp /\ A e. X /\ B e. X ) -> ( A D B ) e. RR* ) $=
+      ( cxms wcel w3a cxp cres co cxr wceq ovres 3adant1 cxmet cfv xmsxmet2
+      xmetcl syl3an1 eqeltrrd ) DHIZAEIZBEIZJABCEEKLZMZABCMZNUEUFUHUIOUDABEECPQ
+      UDUGERSIUEUFUHNICDEFGTABUGEUAUBUC $.
+
+    $( The distance function in an extended metric space is nonnegative.
+       (Contributed by Mario Carneiro, 4-Oct-2015.) $)
+    xmsge0 $p |- ( ( M e. *MetSp /\ A e. X /\ B e. X ) ->
+                   0 <_ ( A D B ) ) $=
+      ( cxms wcel w3a cc0 cxp cres co cle cxmet cfv wbr xmsxmet2 xmetge0 ovres
+      syl3an1 wceq 3adant1 breqtrd ) DHIZAEIZBEIZJKABCEELMZNZABCNZOUFUIEPQIUGUH
+      KUJORCDEFGSABUIETUBUGUHUJUKUCUFABEECUAUDUE $.
+
+    $( The distance function in an extended metric space is symmetric.
```

Comment:
"The value of the distance function in an extended metric space is 0 iff its arguments are identical."
or
"The distance between two points in an extended metric space is 0 iff the two points are identical."

Reply:
I took the second one and spelled out the word "zero" because I guess that is the usual style (although I see we have some of each). Done in both set.mm and iset.mm.